README.md

```diff
@@ -5,6 +5,10 @@ Adds Vim-style shortcuts to [Min](https://github.com/minbrowser/min):
 * Press f to search for links (or F to open the link in a new tab)
 * Press j to scroll down
 * Press k to scroll up
+* Press gg to scroll to the top
+* Press G to scroll to the bottom
+* Press yy to copy the current url
+* Press dd to close the tab
```

Comment:
can you remove this?

Reply:
removed!

Reply:
Actually, I didn't mean to remove all of them, just the "dd" shortcut since that isn't included anymore.

Reply:
ah my bad, fixed.